java/test/jmri/ArchitectureTest.java

```diff
@@ -335,8 +335,9 @@ static public void tearDown() {
         .doNotHaveFullyQualifiedName("jmri.jmrix.powerline.dmx512.SpecificTrafficController").and()
         .doNotHaveFullyQualifiedName("jmri.jmrix.sprog.serialdriver.SerialDriverAdapter").and()
         .doNotHaveFullyQualifiedName("jmri.jmrix.zimo.mx1.Mx1Adapter").and()
-        .doNotHaveFullyQualifiedName("jmri.jmrix.zimo.mxulf.SerialDriverAdapter")
-
+        .doNotHaveFullyQualifiedName("jmri.jmrix.zimo.mxulf.SerialDriverAdapter").and()
+        //the XBee code uses an event driven library, so the code registers as an event listener.
+        .doNotHaveFullyQualifiedName("jmri.jmrix.ieee802154.xbee.XBeeAdapter")
         .should().accessClassesThat().resideInAPackage("com.fazecast.jSerialComm..");
```

Comment:
It would be good with an empty line between these two lines. It was there before this change. It makes the code easier to read.

Reply:
The blank line there is actually inconsistent with most, if not all, of the other methods in this test class.